documentation/asciidoc/computers/raspberry-pi/pcie.adoc

```diff
@@ -0,0 +1,67 @@
+== Raspberry Pi connector for PCIe
+
+.Raspberry Pi connector for PCIe
+image::images/pcie.jpg[alt="Raspberry Pi connector for PCIe",width="70%"]
+
+Raspberry Pi 5 has an FPC connector on the right-hand side of the board. This connector breaks out a PCIe Gen 2.0 ×1 interface for fast peripherals.
+
+For more information about the PCIe FPC connector pinout and other details needed to create third-party devices, accessories, and HATs, see the https://datasheets.raspberrypi.com/pcie/pcie-connector-standard.pdf[Raspberry Pi Connector for PCIe] standards document. It should be read alongside the https://datasheets.raspberrypi.com/hat/hat-plus-specification.pdf[Raspberry Pi HAT+ Specification].
+
+=== Enabling PCIe
+
+By default the PCIe connector is not enabled. To enable it, add the following line to `/boot/firmware/config.txt`:
+
+[source]
+----
+dtparam=pciex1
+----
+
+Reboot with `sudo reboot` for the configuration changes to take effect.
+
+NOTE: You can also use the alias `nvme`.
```

Comment:
Unfortunately I don't have hardware to test this on. But if Tom says it isn't necessary for him, I believe it. I'll remove this bit.

Reply:
So I think the difference is whether we are using the PCIe as the boot disk or not. If we are *booting from an NVMe drive* then this isn't needed. You just need to make sure you have the boot order correct, and a recent firmware.

If you're booting from an SD Card you'll need this, so that PCIe is activated and you can use it as a drive. Alternatively, you'll need it if you're using PCIe for something that isn't NVMe.

Reply:
That's a negative, @aallan. Booting from the SDcard on a Pi5b, I'm testing 'secure' NVMe drives via our M2 HAT+, and have not needed to use this directive to format and generally punish the candidate drives.

Reply:
Ah. If you're using our hardware, then that's a HAT+. The HAT+ configures things for you. If you're not on our hardware you can't guarantee it's a HAT+. So what this section really needs is rewriting from that perspective. I think there's a NOTE block in there somewhere about this, but it needs to be cleaned up and all that put inline.

Reply:
On it.